.github/workflows/release.yml

```diff
@@ -0,0 +1,72 @@
+name: Release
```

Comment:
In theory, this workflow could be combined with `main.yml` but it would complicate some things (like the `concurrency` setting). I think that it should be fine to run it in "parallel" - but I could try to combine them later if you prefer that.